docs/learn-account-restore.md

```diff
@@ -0,0 +1,98 @@
+---
+id: learn-account-restore
+title: Restoring Accounts
+sidebar_label: Restoring Accounts
+description: Step-by-step guides on restoring a Polkadot account.
+---
+
+In the case that your Polkadot account needs to be restored, there are simple ways to take care of
+this in Polkadot.js Browser Plugin, Polkadot.js, and Parity Signer. Note that in order to recover an
+account, it is necessary that an account is made according to the instructions
+[here](learn-account-generation).
+
+## Polkadot.js Browser Plugin
+
+For using the Polkadot.js browser plugin, restoring an account is simple. Let's start off by heading
+over to the extension and selecting the desired account to restore. Click on the three dots beside
```

Comment:
It seems a little strange to me to start this document talking about account restoration, and then go into the process of creating a backup. Maybe replace `to restore` here with `to back up and later restore`.